airbyte-integrations/connectors/source-bigquery/acceptance-test-config.yml

```diff
@@ -0,0 +1,7 @@
+# See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-bigquery:dev
+tests:
+  spec:
+    - spec_path: "src/test-integration/resources/expected_spec.json"
+      config_path: "src/test-integration/resources/dummy_config.json"
```

Comment:
BTW in source python connectors we usually store dummy, invalid configs here:
`integration_tests/invalid_config.json`
it just FYI

Reply:
thanks, this is a valid config, but mocked :)
we don't have any configs actually before we up test environment.